api/providers.go

```diff
@@ -113,3 +118,85 @@ func loadInternalProviders(cfg *config.Config, DCOSTools dcos.Tooler) (internalC
 		HTTPEndpoints: httpEndpoints,
 	}, nil
 }
+
+func LoadCollectors(cfg *config.Config, tools dcos.Tooler, client *http.Client) ([]collectors.Collector, error) {
+	providers, err := loadProviders(cfg, tools)
+	if err != nil {
+		return nil, fmt.Errorf("could not init bundle handler: %s", err)
+	}
+	var coll []collectors.Collector
```

Comment:
since this is a longer function, I think we should use a more descriptive name, like `collectors`